src/app/docs/text-editor/text-editor-docs.component.html

```diff
@@ -20,5 +20,120 @@
+          </p>
+          <sky-docs-design-guideline-thumbnail
+            caption="Use a text editor when users need to manipulate blocks of text."
+            captionType="success"
+            imageSource="~/assets/img/use-when.png"
+          >
+          </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+  
+        <sky-docs-design-guideline
+          heading="Don't use when"
+          headingStyle="danger"
+        >
+          <p class="sky-docs-text-block">
+            Don’t use the text editor when users only need to enter unformatted plain text. Use a text input or textarea instead.
+          </p>
+        </sky-docs-design-guideline>
+      </sky-docs-demo-page-section>
+  
+      <sky-docs-demo-page-section
+        heading="Anatomy"
+      >
+          <sky-docs-anatomy
+            imagePath="~/assets/img/anatomy.png"
+          >
+            <sky-docs-anatomy-item>
+              Label
+            </sky-docs-anatomy-item>
+            <sky-docs-anatomy-item>
+              Menu bar
+            </sky-docs-anatomy-item>
+            <sky-docs-anatomy-item>
+              Toolbar
+            </sky-docs-anatomy-item>
+            <sky-docs-anatomy-item>
+              Text entry field
+            </sky-docs-anatomy-item>
+          </sky-docs-anatomy>
+
+      </sky-docs-demo-page-section>
+  
+      <sky-docs-demo-page-section
+        heading="Options"
+      >
+        <sky-docs-design-guideline
+          heading="Menu bar & toolbar"
+        >
+          <p class="sky-docs-text-block">
+            Default menu items:
+          </p>
+          <ul>
+            <li>Edit</li>
+            <li>Format</li>
+          </ul>
+          <p class="sky-docs-text-block">
+            Optional menu item:
+          </p>
+          <ul>
+            <li>Merge</li>
+          </ul>
+          <p class="sky-docs-text-block">
+            Default toolbar items:
+          </p>
+          <ul>
+            <li>Font</li>
+            <li>Text size</li>
+            <li>Text styles (bold/italic/underline)</li>
+            <li>Text color</li>
+            <li>Highlight color</li>
+            <li>Lists (numbered/bulleted)</li>
+            <li>Hyperlink</li>
+          </ul>
+          <p class="sky-docs-text-block">
+            Optional toolbar items:
+          </p>
+          <ul>
+            <li>Alignment (left/center/right)</li>
+            <li>Undo/redo</li>
+            <li>Indentation</li>
+          </ul>
+          <sky-docs-design-guideline-thumbnail
+            caption="Text editor with default options."
+            imageSource="~/assets/img/option-menu-toolbar-items.png"
+          >
+          </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+      </sky-docs-demo-page-section>
+
+      <sky-docs-demo-page-section
+      heading="Related information"
+    >
```

Comment:
Is it worth adding a Components section here to link to https://developer.blackbaud.com/skyux/components/form? And what about https://developer.blackbaud.com/skyux-forms/docs/input-box?docs-active-tab=design? I'm not sure either way, but just throwing it out to see if those are related docs that consumers should be aware of.

Reply:
i don't think we need related components - the text editor will be pretty different than other inputs